core/src/test/java/com/amplifyframework/logging/AndroidLoggingPluginTest.java

```diff
@@ -131,6 +137,39 @@ public void noContentLoggedAtThresholdNone() {
         assertTrue(systemLog.getLines().isEmpty());
     }
 
+    /**
+     * When the application is not in a debuggable build, logs are not stored.
+     */
+    @Test
+    public void noLogsStored() {
+        Context mockContext = mock(Context.class);
+        ApplicationInfo mockAppInfo = mock(ApplicationInfo.class);
+        mockAppInfo.flags = 0;
+        when(mockContext.getApplicationInfo()).thenReturn(mockAppInfo);
+        AndroidLoggingPlugin plugin = new AndroidLoggingPlugin();
+        plugin.configure(new JSONObject(), mockContext);
+        Logger logger = plugin.forNamespace("logging-test");
+        logger.info("Info log");
+        assertNull(plugin.getLogs());
```

Comment:
Better to return an empty collection than null 
```
assertEquals(Collections.emptyList(), plugin.getLogs())
```

https://www.thefinestartist.com/effective-java/43